bookstore/client/store_client.py

```diff
@@ -7,37 +7,38 @@
 class BookstoreClient(CurrentNotebookClient):
     """Represents a bookstore client that corresponds to the active nb client"""
 
-    def __init__(self, s3_bucket=""):
+    def __init__(self, s3_bucket=None):
         if s3_bucket:
             self.default_bucket = s3_bucket
         super().__init__()
 
     @property
     def publish_endpoint(self):
+        # TODO: refactor to /api/bookstore/publish/ ?
```

Comment:
Agreed. Can be a separate PR.

Reply:
This is specifically a request from @rgbkrk to have this be more REST-ful in its naming. Rather than "publish" the action this is intended to be a reference to a location "published" that refers to the notebooks that are there.

I find it somewhat confusing as well, but I don't want to break from this approach without getting @rgbkrk's sign off.

Reply:
It's probably ok to just refer to it in an RPC way, especially since I'm the only one who thought otherwise. I'd prefer what makes sense to most people.

Reply:
I think at this point that we should leave as is.